src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -38,4 +44,13 @@ public SuburbListGetDto addDutyAreas(@RequestBody @Valid SuburbListPostDto subur
         return franchiseeService.addDutyAreas(suburbListPostDto, franchiseeId);
     }
 
+    @GetMapping("/{id}/orders")
+    public List<OpenOrderResponseDto> getOpenOrders(@PathVariable Long id, @RequestParam @NotNull String status){
+        if ("open".equals(status)) {
```

Comment:
It should be status.equals("open"). And I think this conditional expression should be in the service level

Reply:
this compare actually should use Enum...

Reply:
Well, this method may be extended to support other status, and it's better to check status in Controller.